srcpkgs/python-wsproto/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'python-wsproto'
+pkgname=python-wsproto
+version=0.12.0
+revision=1
+noarch=yes
+wrksrc=wsproto-${version}
+build_style=python-module
+pycompile_module="wsproto"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python python-h11"
+short_desc="Sans-IO WebSocket protocol implementation (Python2)"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://wsproto.readthedocs.io"
+distfiles="https://github.com/python-hyper/wsproto/archive/${version}.tar.gz"
+checksum=a133e55752d29cba690d00d51ecbfec6624d15f7cd0634452d3b03b495c2086c
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-wsproto_package() {
+	noarch=yes
+	pycompile_module="wsproto"
+	depends="${depends/python/python3}"
```

Comment:
please spell it out, don't use variable replacing

Reply:
Done